cluster-bootstrap/00000_namespaces-needed-for-monitoring.yaml

```diff
@@ -28,5 +28,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openshift-etcd
-
-  
\ No newline at end of file
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-ingress
```

Comment:
is this needed for monitoring?  i find the `ingress-operator` creates this namespace.

Reply:
This is mainly to support the creation of the router service below. Yes, I can let the ingress operator create it, but that will only happen after the nodes have joined the cluster. While that happens, the pod that applies these manifests has been crashlooping because it cannot create the router service, and the backoff is long. This helps the service get created quickly and it can sit there until pods are available for it.